libs/mixer/melody.ts

```diff
@@ -110,6 +110,65 @@ namespace music {
 
     let playToneID = 0
 
+    /**
+     * Play a melody from the melody editor.
+     * @param melody - string of up to eight notes [C D E F G A B C5] separated by spaces, 
+     * which will be played one at a time, eg: "E D G F B A C5 B "
```

Comment:
Don't use eg:, it actually has special meaning in our comment parser. The new syntax is to do `melody.defl="whatever"`